pages/common/tar.md

```diff
@@ -6,32 +6,28 @@
 
 - Create an archive from files:
 
-`tar -cf {{target.tar}} {{file1 file2 file3}}`
+`tar cf {{target.tar}} {{file1 file2 file3}}`
 
 - Create a gzipped archive:
 
-`tar -czf {{target.tar.gz}} {{file1 file2 file3}}`
+`tar czf {{target.tar.gz}} {{file1 file2 file3}}`
 
-- Extract an archive in a target directory:
-
-`tar -xf {{source.tar}} -C {{directory}}`
+- Extract a (compressed) archive in the current directory:
```

Comment:
Perhaps this will read better?
```suggestion
- Extract a (compressed) archive into the current directory:
```

Otherwise, it's implying that the compressed archive to extract has to be in the current directory to perform the extraction.

Reply:
Not a new page, just trying to improve upon it. Applied your suggestions.